README.md

```diff
@@ -6,41 +6,184 @@
+
+![](docs/fast_forward.gif)
+
+### Slide out branch
+
+When branch was merged (and sometimes in other cases as well), you probably want to remove this branch from branch tree.
+To do this with easy (and without manual editing of `machete` file) you can use `Slide Out Selected Branch` action.
+This action will modify branch tree for you - delete the selected branch and reattach its children as children of deleted branch parent.
+
+![](docs/slide_out.gif)
+
+![](docs/slide_out2.gif)
+
+### Slide in branch
+
+You can also create new branch bellow selected one (also without manual editing `machete` file). To do this, choose `Slide In Branch Bellow Selected Branch` action from context menu.
+Dialog window will appear.
+In text box type new branch name and click `Slide In` button.
+Another dialog will appear - it is a standard new branch IntelliJ dialog.
+Here you can choose if you want to check out a newly created branch or not.
+
+![](docs/slide_in1.gif)
+
+This action can also be used to reattach an existing branch below a selected one.
+In this case you just must type name of existing branch that you want to reattach and then click `Slide In`.
+You can also select `Reattach children` checkbox to move along all children of the reattached branch.
+
+![](docs/slide_in2.gif)
+
+![](docs/slide_in3.gif)
+
+### Other actions
+
+On the left side bar you can find other actions (from top to bottom):
+- ![](docs/left_bar_actions/refresh.png) **Refresh Status** - refreshing (reload) branch tree (the graph displayed in main plugin window)
+- ![](docs/left_bar_actions/toggle_listing_commits.png) **Toggle Listing Commits** - show or hide commits belonging to branches (for more details see section [Listing commits](#listing-commits))
+- ![](docs/left_bar_actions/open_machete_file.png) **Open Machete File** - open `machete` file in IntelliJ editor (to see what this editor can do see [Machete file editor](#machete-file-editor) section)
+- ![](docs/left_bar_actions/fetch_all_remotes.png) **Fetch All Remotes** - equivalent of `git fetch --all` command
+- ![](docs/left_bar_actions/push_current_branch.png) **Push Current Branch** - push branch that is currently checked out (shortcut of [Push](#push) context menu action for the current branch)
+- ![](docs/left_bar_actions/rebase_current_branch.png) **Rebase Current Branch Onto Parent** - rebase currently checked out branch onto its parent (shortcut of [Rebase](#rebase) context menu action for the current branch)
+- ![](docs/left_bar_actions/help.png) **Show Help Window** - show window with example branch tree and explanation what parts of this graph mean
+
+### Multi-repository support
+
+Git Machete support many git repositories in one project. It can be many standard repositories or submodules.
+If more than one repository will be detected, then selection list will appear in top of main plugin window.
+In that case each repository will have its own `machete` file.
+
+![](docs/multi_repo.gif)
```

Comment:
I think that `Peek` is the only working (on fedora) and the worst screen recorder :c It is impossible to maintain a standard of gifs sizes. This inconsistency makes my eyes bleed :sob: but I know we must leave with it for now

Reply:
I'm not sure what do you mean. GIFs have exactly that size as recorded area.

Reply:
Imagine a world where all gifs have the same size (at least within the single page, file)... :lemon: nvm